carbon/app.py

```diff
@@ -34,20 +49,14 @@ def people():
     sql = select([persons.c.MIT_ID, persons.c.KRB_NAME_UPPERCASE,
                   persons.c.FIRST_NAME, persons.c.MIDDLE_NAME,
                   persons.c.LAST_NAME, persons.c.EMAIL_ADDRESS,
-                  persons.c.ORIGINAL_HIRE_DATE, persons.c.HR_ORG_UNIT_TITLE,
+                  persons.c.ORIGINAL_HIRE_DATE, dlcs.c.HR_ORG_LEVEL4_NAME,
                   persons.c.PERSONNEL_SUBAREA_CODE, orcids.c.ORCID]) \
         .select_from(persons.outerjoin(orcids)) \
-        .where(persons.c.EMAIL_ADDRESS != None) \
```

Comment:
Since the email field is required by Elements, we should filter out 
records that do not have one.

I don't know if it would throw an error and abort the data feed import 
or just ignore the record and continue with the import but I agree we 
should avoid the noise either way.

Tim


On 3/23/2016 8:21 PM, Mike Graves wrote:
>
> In carbon/app.py 
> <https://github.com/MITLibraries/carbon/pull/20#discussion_r57259063>:
>
> >                    persons.c.PERSONNEL_SUBAREA_CODE, orcids.c.ORCID]) \
> >          .select_from(persons.outerjoin(orcids)) \
> > -        .where(persons.c.EMAIL_ADDRESS != None) \
>
> To clarify, my understanding is that if a person doesn't have an email 
> address, Elements will skip it. The end result will be the same if we 
> include people without an email address, but I'd rather just get rid 
> of the noise at the start.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/MITLibraries/carbon/pull/20/files/0997d0a9cb5dd369fa9ea62baeb3ae3a719d4082#r57259063>
>